src/core/openthread-core-default-config.h

```diff
@@ -675,7 +679,7 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_NCP_UART_RX_BUFFER_SIZE
-#define OPENTHREAD_CONFIG_NCP_UART_RX_BUFFER_SIZE               1300
+#define OPENTHREAD_CONFIG_NCP_UART_RX_BUFFER_SIZE               512
```

Comment:
This seems unrelated to this commit.... Any reason to reduce the uart/spi buffer sizes?